pages/agent/v3/help/_env_get.md

```diff
@@ -17,57 +17,65 @@ script.
 `buildkite-agent env get [variables]`
 
 ### Description
-Retrieves environment variables and their current values from the current job execution environment.
+Retrieves environment variables and their current values from the current job
+execution environment.
```

Comment:
This is what the script outputs. I don't think it makes a difference to the HTML:
![2023-08-15T22:12:59,775838047+10:00](https://github.com/buildkite/docs/assets/11096602/4903ecac-a350-4a59-bf33-0f0d6cfa67c9)
In the agent, we need these line breaks to fit the help text into a terminal window.